sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -125,7 +129,7 @@ case class DescribeFunctionCommand(
 
           if (isExtended) {
             result :+
-              Row(s"Extended Usage:\n${replaceFunctionName(info.getExtended, info.getName)}")
+              Row(s"Extended Usage:${replaceFunctionName(info.getExtended, info.getName)}")
```

Comment:
Yes, it seems `stripMargin` not working in annotations in 2.11 too. In the comments, it is about doing `stripMargin` actually right before printing (not trying to assign the string with `stripMargin`).